Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -537,6 +537,8 @@ class BrowserViewController: UIViewController {
         if !displayedRestoreTabsAlert && hasPendingCrashReport() {
             displayedRestoreTabsAlert = true
             showRestoreTabsAlert()
+            log.debug("Displayed alert to restore tabs after a crash.")
+            SentryIntegration.shared.send(message: "Displayed alert to restore tabs after a crash.", tag: "BrowserDB", severity: .info)
```

Comment:
You can catch and report the crash from TabManger instead. You'll be able to track what caused the crash too.